py-reproducibility.Rmd

```diff
@@ -3,14 +3,33 @@
 ```{r py-reproducibility-setup, include=FALSE}
 source("_common.R")
 ```
+In general reproducibility means that your data and code are available and readable to fully rerun the analysis and produce the final results of the project (e.g. figures in a publication). Reproducibility brings a level of transparency to your research methodology and results that strengthens your arguments, because other researchers can confirm your findings (equally helpful if you are reporting positive or negative results). Establishing reproducibility principles for your projects benefits your future self with saving your time to reproduce or build on the preivous work.  The main elements of reproducibility are 
```

Comment:
```suggestion
In general reproducibility means that your data and code are available and readable to fully rerun the analysis and produce the final results of the project (e.g. figures in a publication). Reproducibility brings a level of transparency to your research methodology and results that strengthens your arguments, because other researchers can confirm your findings (equally helpful if you are reporting positive or negative results). Establishing reproducibility principles for your projects benefits your future self with saving your time to reproduce or build on the preivous work.  The main elements of reproducibility are:
 
```